src/compiler/commandLineParser.ts

```diff
@@ -361,6 +395,63 @@ namespace ts {
         return optionNameMapCache;
     }
 
+    let libOptionNameArrayCache: string[];
+

```

Comment:
i do not think we need to special case this one. we already have rootDirs that is a list and we special case it. we should just handle a `list` type as a valid commandline option type.